test/iron-menu-behavior.html

```diff
@@ -333,177 +365,197 @@
+              MockInteractions.keyDownOn(menu, 85);
 
-            // Key press 'g'
-            MockInteractions.keyDownOn(menu, 71);
+              // Key press 'g'
+              MockInteractions.keyDownOn(menu, 71);
 
-            Polymer.Base.async(function() {
-              var focusedItem = Polymer.dom(menu).node.focusedItem;
-              assert.equal(focusedItem, menu.items[1], 'menu.items[1] is focused');
-              done();
+              Polymer.Base.async(function() {
+                var focusedItem = Polymer.dom(menu).node.focusedItem;
+                assert.equal(focusedItem, menu.items[1], 'menu.items[1] is focused');
+                done();
+              });
             });
           });
         });
 
         test('focusing on item with bogus attr-for-item-title', function(done) {
           var menu = fixture('bogus-attr-for-item-title');
-          MockInteractions.focus(menu);
-
-          // Wait for async focus
-          Polymer.Base.async(function() {
-            // Key press 'i'
-            MockInteractions.keyDownOn(menu, 73);
+          waitForIronItemsChanged(menu, function() {
+            MockInteractions.focus(menu);
 
+            // Wait for async focus
             Polymer.Base.async(function() {
-              var focusedItem = Polymer.dom(menu).node.focusedItem;
-              assert.equal(focusedItem, menu.items[0], 'menu.items[0] is still focused');
-              done();
+              // Key press 'i'
+              MockInteractions.keyDownOn(menu, 73);
+
+              Polymer.Base.async(function() {
+                var focusedItem = Polymer.dom(menu).node.focusedItem;
+                assert.equal(focusedItem, menu.items[0], 'menu.items[0] is still focused');
+                done();
+              });
             });
           });
-
         });
 
         test('focusing non-item content does not auto-focus an item', function(done) {
           var menu = fixture('basic');
-          menu.extraContent.focus();
-          Polymer.Base.async(function() {
-            var menuOwnerRoot = Polymer.dom(menu.extraContent).getOwnerRoot() || document;
-            var menuActiveElement = Polymer.dom(menuOwnerRoot).activeElement;
-            assert.equal(menuActiveElement, menu.extraContent, 'menu.extraContent is focused');
-            assert.equal(Polymer.dom(document).activeElement, menu, 'menu is document.activeElement');
-            done();
+          waitForIronItemsChanged(menu, function() {
+            menu.extraContent.focus();
+            Polymer.Base.async(function() {
+              var menuOwnerRoot = Polymer.dom(menu.extraContent).getOwnerRoot() || document;
+              var menuActiveElement = Polymer.dom(menuOwnerRoot).activeElement;
+              assert.equal(menuActiveElement, menu.extraContent, 'menu.extraContent is focused');
+              assert.equal(document._activeElement || Polymer.dom(document).activeElement, menu, 'menu is document.activeElement');
```

Comment:
Hmm, I need to take another look at this section. I think `_activeElement` might have been removed?